### PR TITLE
FIX: Safari issue with some emojis

### PR DESCRIPTION
**Pull request by pmusaraj**

### CODE REVIEW

File: lib/pretty_text/shims.js [hunk 1]
```diff
@@ -19,6 +19,9 @@ __setUnicode = function(replacements) {
         replacement = "\u200b" + replacement;
       }
       text = text.replace(m[0], replacement);
+      // fixes Safari VARIATION SELECTOR-16 issue with some emojis
+      // https://meta.discourse.org/t/emojis-selected-on-ios-displaying-additional-rectangles/86132
+      text = text.replace('\ufe0f', '');
```
Comment (ZogStriP) [inline lib/pretty_text/shims.js]: Do we want to run that in the loop? Can't it be done once before the loop?

Comment (pmusaraj) [reply lib/pretty_text/shims.js]: You're of course right, I moved it outside the loop.

**Comment by tgxworld** [comment]
I think we should add a spec for this in https://github.com/discourse/discourse/blob/43512e9865087d10fb08a814e8f75f8b765aab6d/spec/components/pretty_text_spec.rb

**Comment by pmusaraj** [comment]
Sure thing, done!

> I think we should add a spec for this in https://github.com/discourse/discourse/blob/43512e9865087d10fb08a814e8f75f8b765aab6d/spec/components/pretty_text_spec.rb

Added the spec.

**Comment by tgxworld** [comment]
@pmusaraj Nice :+1: :heart_eyes_cat:

**Pull request merged**